javaslang/src-gen/main/java/javaslang/Tuple3.java

```diff
@@ -124,6 +124,17 @@ public T1 _1() {
     }
 
     /**
+     * Wither of the 1st element of this tuple.
+     *
+     * Note: The wither is NOT named {@code _1} due to possible ambiguity when using method reference (Tuple3::_1).
+     *
+     * @return a copy of this tuple with a new value for the 1st element of this Tuple.
+     */
+    public Tuple3<T1, T2, T3> w1(T1 t1) {
```

Comment:
I think they're too short and quite confusing, I have to read the doc to understand what they do.